### PR TITLE
WRO-3290: Fixed WizardPanels to support hiding subtitle area.

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
We need to support new `WizardPanels` style without subtitle area.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added `noSubtitle` prop in `WizardPanels` to hide subtitle area.
Added `compactWizard` type in `Panels.Header` component.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-3290

### CODE REVIEW

File: samples/sampler/stories/qa/WizardPanels.js [hunk 1]
```diff
@@ -360,3 +360,54 @@ _WizardPanelsWithAlert.parameters = {
 		noPanel: true
 	}
 };
+
+
```
Comment (jeonghee27) [inline samples/sampler/stories/qa/WizardPanels.js:364]:
```suggestion

```

File: Panels/Header.js [hunk 2]
```diff
@@ -370,12 +370,12 @@ const HeaderBase = kind({
...
 
 			// WizardPanels uses an animated title but that isn't supported for other types
-			if (arranger && type === 'wizard') {
+			if (arranger && (type === 'wizard' || type === 'compactWizard')) {
 				return (
 					<Cell className={css.titleCell} component={ViewManager} arranger={arranger} duration={500} index={0}>
 						<div className={css.titleContainer} key={title + subtitle}>
 							{titleHeading}
-							{subtitleHeading}
+							{type === 'compactWizard' ? '' : subtitleHeading}
```
Comment (jeonghee27) [inline Panels/Header.js:378]:
```suggestion
							{noSubtitle ? null : subtitleHeading}
```

File: samples/sampler/stories/qa/WizardPanels.js [hunk 3]
```diff
@@ -360,3 +360,54 @@ _WizardPanelsWithAlert.parameters = {
...
+export const WizardPanelsWithoutSubtitle = (args) => (
+	<WizardPanels
+		current={args['current']}
+		noAnimation={args['noAnimation']}
+		noSteps={args['noSteps']}
+		noSubtitle={args['noSubtitle']}
```
Comment (jeonghee27) [inline samples/sampler/stories/qa/WizardPanels.js:370]: Please add this into default sampler (WizardPanels / Panels.header)

Comment (jeonghee27) [reply samples/sampler/stories/qa/WizardPanels.js:370]: I don' think we need qa-sampler

Comment (juwonjeong) [reply samples/sampler/stories/qa/WizardPanels.js]: It's removed and moved to default sampler

File: Panels/Header.js (hunk 2 above)
Comment (juwonjeong) [reply Panels/Header.js]: It's fixed.

File: samples/sampler/stories/qa/WizardPanels.js (hunk 1 above)
Comment (juwonjeong) [reply samples/sampler/stories/qa/WizardPanels.js]: It's fixed.

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Pull request merged**